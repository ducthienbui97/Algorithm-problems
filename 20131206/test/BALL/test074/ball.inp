8 800526 950612 670101 266602
890692 426425 858977 602810 575725 111425 74948 914574 956748 214514 921989 705631 66452 528555 326451 626993 597497 25807 875540 461042 885914 47995 495677 792351 812804 139154 232048 461406 984283 840572 889680 74802 884745 522534 159551 179062 851586 287945 89768 141533 660717 830890 548601 627302 334116 18438 557 878987 165183 629810 409675 445713 681175 350753 830344 998511 342269 803533 601622 222127 143071 332040 678030 191111 
831325 350338 348890 5640 727360 10806 252902 429876 6266 1517 165565 874653 365426 286875 371694 535091 421720 474883 635214 348020 496559 404963 976364 136532 149024 576181 355498 859874 231060 728515 182040 559139 543908 96813 550146 954897 878801 152036 786039 46309 80618 439503 859718 790340 260974 187068 817775 413724 651583 530263 723838 872859 226741 438174 458748 830254 319007 501408 926362 244324 138261 504846 703843 826361 
737302 934173 639667 697526 539831 234472 759423 249685 841076 4171 507141 671967 436723 538941 550406 207168 810718 781730 854100 780924 668761 615276 607789 224790 832825 966372 117114 343769 522934 464288 74123 967740 413756 498666 945382 244269 260434 25116 983328 62320 210002 927535 218853 849414 262987 69363 539673 8189 765588 552012 760166 132818 502243 541736 81902 987547 632577 844455 836382 405926 713846 222026 718595 968898 
405444 870596 959165 469852 689811 454221 707222 652836 768686 545615 110887 328132 751715 354762 920959 491538 56174 277599 800549 144336 301486 47730 597004 131301 622266 215169 588806 918248 769143 307841 919171 327143 282317 9576 402246 959529 356796 134170 112739 814629 239555 576188 285155 98748 508085 152520 520804 815896 479534 991397 880733 676555 667804 473977 290709 548712 755046 507121 94147 234749 126489 839186 596671 187385 404015 762263 538415 898711 88799 133546 715536 485678 313222 15189 960451 190432 207249 972781 691177 243740 941957 43421 801923 365084 814840 794882 836370 941938 714302 67722 812816 191147 828637 250058 941637 712220 52680 874536 878529 239841 166206 722743 531627 430686 756971 684253 404883 848982 629206 415605 186510 329801 801743 6935 742524 924824 400362 507200 577765 695038 688539 739080 630701 152021 792901 727573 959488 231351 198189 846634 994875 163348 654913 481972 350460 680086 421528 879659 543628 10850 275583 316009 867807 930333 777218 312046 737471 199114 90772 449728 249858 501646 713931 186628 95208 722878 861952 438277 159870 424543 927743 552473 715671 835370 123340 269865 388852 799323 315116 126839 221611 778241 321672 145044 509888 284855 483859 633954 867258 748455 74349 332127 671482 868570 606362 735744 685657 244890 761988 761186 601224 900526 552496 356850 54541 812946 206410 305740 802083 759304 96925 789572 432541 404683 531382 485346 551296 80396 703956 354461 915564 286292 232559 759971 653274 740853 727539 477452 786829 474884 117447 744971 181611 665114 302023 312566 486308 146590 907340 710528 106641 779085 172392 374779 301246 697955 638230 522969 115088 377297 8800 955219 787293 637846 698757 295709 837414 786555 441735 558386 540682 978606 368 558677 971902 614109 910236 132784 273997 400957 3010 400812 79634 439228 149488 304612 169196 299315 767525 124111 978993 233484 760596 64209 392805 142505 89132 18411 728032 620144 842121 397292 557341 198791 866306 412761 594600 48338 197387 751640 64315 472619 761453 447559 999133 609777 279645 192390 636276 884225 95967 804075 476093 817037 823157 878917 505631 14016 647436 89172 415054 824605 105014 657281 449307 110955 306940 573608 758732 803594 937649 601350 87034 716394 741604 95962 762909 388732 312084 18208 59038 514520 486595 625805 28677 406616 131376 431929 820049 559521 908324 974053 403156 656979 702862 715890 11832 326927 845377 407550 833275 48372 433317 395614 211377 706058 864145 292792 493943 9658 722990 242144 757451 182225 536366 39689 432682 605095 694191 252592 268166 992671 160102 53753 206609 805076 27535 299320 31875 837195 665839 977103 970149 916817 747094 37884 645509 907031 172064 66558 885646 223941 361203 106185 59955 528058 53753 770715 983768 455087 667134 249747 278830 148804 553008 439215 386792 884002 914678 368012 234882 840879 965006 219595 611865 900701 102352 313098 416957 607433 145441 925326 68634 230428 155492 661001 575504 363286 70479 252464 792309 45172 702929 585106 419166 728895 715990 476112 597287 961222 399970 221434 829559 403839 491444 435793 529198 508721 728393 162420 345621 481185 754443 897605 324723 79398 919182 585981 800732 446963 553474 486483 122474 702171 385081 114909 220011 294437 931102 945617 412078 15622 919641 220199 247719 837555 930577 519866 942806 736360 208078 678937 988843 253571 430397 644608 410642 75136 645025 104259 617882 368592 171502 958889 761984 880135 957784 256579 127414 727792 92486 273856 107416 419111 116186 746476 879950 737887 576521 121870 319114 252447 