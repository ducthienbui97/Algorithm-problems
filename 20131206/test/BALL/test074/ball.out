16541856699558
